build.js

```diff
@@ -269,7 +268,7 @@ process.nextTick(function () {
         fs.writeFileSync(fpath, code);
         var output;
         try {
-          output = "" + child_process.execSync('node_modules/closurecompiler/bin/ccjs ' +
+          output = "" + child_process.execSync('./node_modules/.bin/google-closure-compiler-js ' +
```

Comment:
`process.cwd()`, i think, since all commands should be run from project root (assuming both work)